### PR TITLE
In preparation for next ykman cli release: Bumped python and minor changes to actions

### PR DESCRIPTION
**Pull request by fdennis**

### CODE REVIEW

**Comment by dagheyman** [comment]
Looks good, is it on purpose to move the copying of the .DLLs to after `windeployqt` ?

**Comment by fdennis** [comment]
> Looks good, is it on purpose to move the copying of the .DLLs to after `windeployqt` ?

Yes. There was a reason for doing so, but I'm pretty sure it doesn't apply anymore. Will change it back. Good catch!

File: .github/workflows/windows-x64.yml [hunk 1]
```diff
@@ -105,7 +109,7 @@ jobs:
 
       - name: Create an unsigned .msi installer package
         run: |
-          $env:PATH += ";$env:WIX\bin"
+          $env:PATH +=";$env:WIX\bin"
```
Comment (emlun) [inline .github/workflows/windows-x64.yml]: While you're at it, can you restore this space too? :smile:

**Comment by emlun** [comment]
> There was a reason for doing so, but I'm pretty sure it doesn't apply anymore.

Looks like the reason was that the `.\release\pymodules\ykman\native` directory doesn't exist before the windeployqt step?

**Comment by emlun** [comment]
Shall we pull this in too? 51b9bd90196bff0ac9b08e10b424b8b94666a4b1

**Comment by fdennis** [comment]
> Shall we pull this in too? [e9d0833](https://github.com/Yubico/yubioath-desktop/commit/e9d0833e8df3b16853b01364f58df1b043dce066)

Yes! Maybe make a PR to the dll-test-auth branch?

**Comment by emlun** [comment]
I'll just push it in :)

File: .github/workflows/windows-x64.yml (hunk 1 above)
Comment (fdennis) [reply .github/workflows/windows-x64.yml]: Restored all the spaces :)

**Pull request merged**